docs/update/update-cluster-pod.md

```diff
@@ -0,0 +1,112 @@
+# Update k0smotron in Cluster API integrated cluster
+
+To update k0smotron cluster deployed with Cluster API, you need to update
+the k0s version and machine names in the YAML configuration file.
+
+1. Localize the configuration of deployed k0smotron cluster in your repository. For example:
+
+    ```yaml 
+    apiVersion: cluster.x-k8s.io/v1beta1
+    kind: Cluster
+    metadata:
+      name: docker-test
+      namespace: default
+    spec:
+      clusterNetwork:
+        pods:
+          cidrBlocks:
+          - 192.168.0.0/16
+        serviceDomain: cluster.local
+        services:
+          cidrBlocks:
+          - 10.128.0.0/12
+      controlPlaneRef:
+        apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+        kind: K0smotronControlPlane
+        name: docker-test-cp
+      infrastructureRef:
+        apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+        kind: DockerCluster
+        name: docker-test
+    ---
+    apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+    kind: K0smotronControlPlane
+    metadata:
+      name: docker-test-cp
+    spec:
+      version: v1.27.2-k0s.0
+    ```
+2. Configure [persistence](https://docs.k0smotron.io/stable/resource-reference/#clusterspecpersistence)
+to prevent data loss. For example:
+
+   ```yaml
+    ---
+    apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+    kind: K0smotronControlPlane
+    metadata:
+      name: docker-test-cp
+    spec:
+      version: v1.27.2-k0s.0
+      persistence:
+        type: hostPath
+        hostPath: "/tmp/kmc-test" # k0smotron will mount a basic hostPath volume to avoid data loss.
+   ```
+
+   Do not configure `hostPath` persistence in production environment. 
```

Comment:
Using the `hostPath` volume type introduces many security risks. Avoid configuring persistence for volumes of the `hostPath` type.